content/applications/marketing/marketing_automation/getting_started/testing_running.rst

```diff
@@ -2,33 +2,45 @@
-which reveals that campaign's detail form.
+Testing campaigns
+=================
 
-On the campaign detail form, make sure the campaign already has activities configured (or build a
-campaign by following the directions here on :doc:`this documentation
+To test a marketing campaign, open the :menuselection:`Marketing Automation` application, and select
+the desired campaign to test, which reveals that campaign's detail form.
+
+On the campaign detail form, make sure the campaign already has activities configured in the
+workflow (or build a campaign by following the directions on :doc:`this documentation
 </applications/marketing/marketing_automation/getting_started/workflow_activities>`).
 
-To start a test, click the :guilabel:`Launch a Test` button at the top of the campaign form.
+To start a test, click the :guilabel:`Launch a Test` button at the top of the campaign form, to the
+right of the :guilabel:`Start` button.
 
 .. image:: testing_running/launch-test.png
    :align: center
-   :alt: Launch a test button in Odoo Marketing Automation.
+   :alt: Launch a test button on a campaign detail form in Odoo Marketing Automation.
+
+When clicked, a :guilabel:`Launch a test` pop-up window appears.
+
+.. image:: testing_running/launch-test-popup-window.png
+   :align: center
+   :alt: Launch a test pop-up window that appears in Odoo Markting Automation.
 
-When clicked, a :guilabel:`Launch a test` pop-up window appears. In the drop-down field, choose an
-existing record (or create a new one) to run the test on. To create a brand new record, click the
-:guilabel:`Search More...` link at the bottom of the drop-down menu, and then click the
-:guilabel:`Create` button.
+Click the field to reveal a drop-down menu of contacts. From this drop-down menu, select an existing
+contact (or create a new one) to run the test on.
 
-Doing so reveals a blank contact form, in which the :guilabel:`Name` and :guilabel:`Email` **must**
-be entered. When all the necessary information has been entered, click :guilabel:`Save`, and Odoo
-returns to the :guilabel:`Launch a test` pop-up, with that new record in the :guilabel:`Choose an
-existing record or create a new one to test` field.
+To create a new contact directly from the :guilabel:`Launch a test` pop-up window, start typing the
+name of the new contact in the blank field, and click :guilabel:`Create and edit...`. Doing so
+reveals a blank :guilabel:`Create Record` pop-up form, in which the necessary contact information
+(:guilabel:`Email`, :guilabel:`Mobile`, etc.) *must* be entered, in order for the test to work.
 
-Once a record is selected, click :guilabel:`Continue`, and Odoo reveals the campaign test page.
+When all the necessary fields have been entered, click :guilabel:`Save & Close` to return to the
+:guilabel:`Launch a test` pop-up window.
+
+Once a contact is selected, click :guilabel:`Launch` to reveal the campaign test page.
 
 .. image:: testing_running/test-screen.png
    :align: center
```

Comment:
You have two very similar screenshots one after the other with not arrows or red-outlines. I found this to be confusing.  @lines 63-65

Reply:
I completely understand. I added a red arrow indicator for the child activity image - I believe that'll be sufficient enough to separate the two. Great note!